apps/site/components/MDX/Calendar/UpcomingMeetings.tsx

```diff
@@ -12,7 +12,7 @@ import styles from './calendar.module.css';
 type GroupedEntries = Record<string, Array<CalendarEvent>>;
 
 const UpcomingMeetings: FC = async () => {
-  const events = await getCalendarEvents(CALENDAR_NODEJS_ID);
+  const events = (await getCalendarEvents(CALENDAR_NODEJS_ID)) ?? [];
```

Comment:
Instead of this, could you update the `getCalendarEvents` function to handle the undefined scenario?

Reply:
Sure, done